src/internal/puck/components/ThemeSidebar.tsx

```diff
@@ -1,7 +1,51 @@
+import { AutoFieldPrivate } from "@measured/puck";
 import React from "react";
+import { ThemeConfig } from "../../../utils/themeResolver.ts";
+import {
+  constructThemePuckFields,
+  constructThemePuckValues,
+} from "../../utils/constructThemePuckFields.ts";
 
-const ThemeSidebar = () => {
-  return <div></div>;
+type ThemeSidebarProps = {
+  savedThemeValues: any;
+  themeConfig?: ThemeConfig;
+  saveTheme: (theme: ThemeConfig) => void;
+};
+
+const ThemeSidebar = (props: ThemeSidebarProps) => {
+  const { saveTheme, themeConfig, savedThemeValues } = props;
+  if (!themeConfig) {
+    return <></>;
```

Comment:
Similar styling to my below comment, add text
> Please contact your developer to set up theme management.

The text will probably change but we can use this for now.